campaigns/rustreach/how-I-came-to-rust.md

```diff
@@ -0,0 +1,58 @@
+---
+title: How I came to Rust
+---
+
+# How I came to Rust
+
+The #rustreach program asked people to write down how they came to Rust and I'm happy to do that. Entering a community or a space is always a moment of orientation process and I hope I can give some insight into that.
+
+My path is highly individual, but yours will be, too. Own that. It's great.
+
+## Picking up Rust
+
+I am certainly an early adopter of Rust. I picked it up as a hobby in the early versions, Rust 0.4 was the first version I tried out and around version 0.6, I started writing notable pieces of software in it. That was in 2013. Rust - for me - was the chance to see a language being built. I had no interested in writing a programming language, but I was curious how the process works. I'm still amazsed on how agressively languages can change when compatibilty is no issue. Back then, _all Rust code in existence_ would break _regularly_.
+
+But there were two things I found very notable:
+
+* The openness of the discussion and the ease of participation
+* The air around the community to build something to _use_
+
+Rust was always the attempt to apply techniques instead of coming up with their own (obviously, the process means that you need to come up with your own things). It was interesting to watch the process and rare to see that. I enjoyed watching and coding along.
+
+## Rust Berlin
+
+In 2014, I got in touch with @johannh about running a meetup. We were both curious about Rust, but wanted to get in touch with other people who were interested. So we formed Rust Berlin and quickly thereafter the Rust Hack & Learn, where - instead of coming and listening to a talk - you could pass by and hack on your own project. The Hack & Learn is still running every other week.
```

Comment:
Perhaps you could make "Rust Hack & Learn" link to a website if it has a specific one?

Reply:
Ah same comment with RustFest below :)